frontend/src/metabase/lib/core.js

```diff
@@ -223,6 +223,11 @@ export const field_special_types = [
     name: t`Updated timestamp`,
     section: t`Date and Time`,
   },
+  {
+    id: TYPE.TextDate,
+    name: t`Text as a timestamp`,
+    section: t`Date and Time`,
```

Comment:
If it's a date and time then don't call it `TextDate`, call it `TextDateTime`. A `TextDate` in my mind would be `2020-11-23`